### PR TITLE
toggle component complete

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] Complete toggle component
image

### CODE REVIEW

File: src/common/components/Toggle/Toggle.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+
+import Toggle, { TToggleProps } from './Toggle'
```
Comment (ArthurVogelsang) [inline src/common/components/Toggle/Toggle.stories.tsx:4]: @sanjanadhikari1 it's good to import from index.ts instead of Toggle.tsx

Comment (sanjanadhikari1) [reply src/common/components/Toggle/Toggle.stories.tsx:4]: yes correct, will fix it asap. thanks for commenting

File: src/common/components/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  toggleHandler,
+  classNames = 'w-34px h-5 rounded-full',
+  selectedClass = 'bg-green-68',
+}) => {
+  const [checked, setChecked] = useState(false)
+  return (
+    <div>
+      <label className='flex items-center cursor-pointer'>
+        <div className='relative'>
+          <input type='checkbox' id='toggleB' className='sr-only' />
```
Comment (ArthurVogelsang) [inline src/common/components/Toggle/Toggle.tsx:21]: @sanjanadhikari1 input tag is needed?

File: src/common/components/Toggle/Toggle.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+
+export type TToggleProps = {
+  classNames?: string
+  selectedClass?: string
+  toggleHandler: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void
+}
+
```
Comment (ArthurVogelsang) [inline src/common/components/Toggle/Toggle.tsx:9]: @sanjanadhikari1 I think we need two props are reqired
1. selected/checked -> boolean 
2. toogleHandler -> selected flag should pass

Without these, how to pass selected flag to pass `toggle` component?
How to know On/Off state from togglehandler func?

File: src/common/components/Toggle/Toggle.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const Template: Story<TToggleProps> = ({ ...args }: TToggleProps) => {
+  return <Toggle {...args} />
+}
+
+export const SimpleToggle = Template.bind({})
+SimpleToggle.args = {
+  classNames: 'w-34px h-5 rounded-full',
+  selectedClass: 'bg-green-68',
+  toggleHandler: e => {
+    console.log(e)
```
Comment (ArthurVogelsang) [inline src/common/components/Toggle/Toggle.stories.tsx:20]: @sanjanadhikari1 Please remove console.log, should be correct handler

File: src/common/components/Toggle/Toggle.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <div className='relative'>
+          <div
+            className={cn(
+              'block bg-gray-dd flex items-center',
+              classNames,
+              checked && selectedClass,
+            )}
+            onClick={() => {
+              toggleHandler(checked)
+              setChecked(!checked)
```
Comment (ArthurVogelsang) [inline src/common/components/Toggle/Toggle.tsx:31]: @sanjanadhikari1 is it correct? one is checked, the other is !checked, I think both should be !checked.
Or you can remove setChecked func, we can remove it.

Comment (sanjanadhikari1) [reply src/common/components/Toggle/Toggle.tsx:31]: yes correct both should be !checked. Thanks for letting me know

Comment (ArthurVogelsang) [reply src/common/components/Toggle/Toggle.tsx:31]: sounds good

File: src/common/components/Toggle/Toggle.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+            }}
+          >
+            <div
+              className={cn(
+                'dot absolute bg-white w-3 h-3 rounded-full',
+                !checked && 'left-1',
+                checked && 'right-1',
```
Comment (reacteer) [inline src/common/components/Toggle/Toggle.tsx:38]: You can simply use ternary operator.
checked ? 'right-1' : 'left-1',

**Review by ArthurVogelsang — APPROVED** [review]

**Review by ArthurVogelsang — CHANGES_REQUESTED** [review]
@sanjanadhikari1 Please fix conflicts

**Review by ArthurVogelsang — APPROVED** [review]

**Review by CrazyOptimist — APPROVED** [review]
Looks good to me.

**Pull request merged**